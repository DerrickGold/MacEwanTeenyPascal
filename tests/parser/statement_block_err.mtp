begin

	(*missing begin statement*)

	(*should complain about extra end statement instead of period*)
	end
end
.