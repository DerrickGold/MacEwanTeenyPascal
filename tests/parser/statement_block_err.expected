Syntax Error: line 7:
Unexpected token (TOK_KEY_END, 'end'), expected a token type of:  
  'TOK_PERIOD'
